<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutToolboxWidget</class>
 <widget class="QWidget" name="LayoutToolboxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mixing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="resizeBox">
     <property name="title">
      <string>Resize</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="sizeHorizontalButton">
        <property name="toolTip">
         <string>Resize selected sources to equal widths (max)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/size-horizontal.png</normaloff>:/glmixer/icons/size-horizontal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="sizeVerticalButton">
        <property name="toolTip">
         <string>Resize selected sources to equal heights (max)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/size-vertical.png</normaloff>:/glmixer/icons/size-vertical.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="sizeOriginalAspectButton">
        <property name="toolTip">
         <string>Restore original aspect ratio</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/size-ratio-original.png</normaloff>:/glmixer/icons/size-ratio-original.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="sizeHorizontalFrameButton">
        <property name="toolTip">
         <string>Resize to output width</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/size-horizontal-frame.png</normaloff>:/glmixer/icons/size-horizontal-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="sizeVerticalFrameButton">
        <property name="toolTip">
         <string>Resize to output height</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/size-vertical-frame.png</normaloff>:/glmixer/icons/size-vertical-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="sizeRenderingAspectButton">
        <property name="toolTip">
         <string>Adjust aspect ratio to match output window</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/size-ratio-frame.png</normaloff>:/glmixer/icons/size-ratio-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rotateBox">
     <property name="title">
      <string>Rotate and flip</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="rotateLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QToolButton" name="rotateClockwiseButton">
        <property name="toolTip">
         <string>Rotate 90 degrees clockwise</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/rotate-90-clockwise.png</normaloff>:/glmixer/icons/rotate-90-clockwise.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="rotateCounterclockwiseButton">
        <property name="toolTip">
         <string>Rotate 90 degrees counterclockwise</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/rotate-90-counterclockwise.png</normaloff>:/glmixer/icons/rotate-90-counterclockwise.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="flipHorizontalButton">
        <property name="toolTip">
         <string>Flip horizontally</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/flip-horizontal.png</normaloff>:/glmixer/icons/flip-horizontal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="flipVerticalButton">
        <property name="toolTip">
         <string>Flip vertically</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/flip-vertical.png</normaloff>:/glmixer/icons/flip-vertical.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="alignBox">
     <property name="title">
      <string>Align</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="alignLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="alignHorizontalLeftButton">
        <property name="toolTip">
         <string>Align to the left border</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-horizontal-left.png</normaloff>:/glmixer/icons/align-horizontal-left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="alignHorizontalCenterButton">
        <property name="toolTip">
         <string>Center horizontally</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-horizontal-center.png</normaloff>:/glmixer/icons/align-horizontal-center.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="alignHorizontalRightButton">
        <property name="toolTip">
         <string>Align to the right border</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-horizontal-right.png</normaloff>:/glmixer/icons/align-horizontal-right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="alignVerticalBottomButton">
        <property name="toolTip">
         <string>Align to the bottom border</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-vertical-bottom.png</normaloff>:/glmixer/icons/align-vertical-bottom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="alignVerticalCenterButton">
        <property name="toolTip">
         <string>Center vertically</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-vertical-center.png</normaloff>:/glmixer/icons/align-vertical-center.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="alignVerticalTopButton">
        <property name="toolTip">
         <string>Align to the top border</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-vertical-top.png</normaloff>:/glmixer/icons/align-vertical-top.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="alignHorizontalLeftFrameButton">
        <property name="toolTip">
         <string>Align to the left border of the frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-horizontal-left-frame.png</normaloff>:/glmixer/icons/align-horizontal-left-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="alignHorizontalCenterFrameButton">
        <property name="toolTip">
         <string>Center horizontally on the frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-horizontal-center-frame.png</normaloff>:/glmixer/icons/align-horizontal-center-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="alignHorizontalRightFrameButton">
        <property name="toolTip">
         <string>Align to the right border of the frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-horizontal-right-frame.png</normaloff>:/glmixer/icons/align-horizontal-right-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="alignVerticalBottomFrameButton">
        <property name="toolTip">
         <string>Align to the bottom border of the frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-vertical-bottom-frame.png</normaloff>:/glmixer/icons/align-vertical-bottom-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="alignVerticalCenterFrameButton">
        <property name="toolTip">
         <string>Center vertically on the frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-vertical-center-frame.png</normaloff>:/glmixer/icons/align-vertical-center-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="alignVerticalTopFrameButton">
        <property name="toolTip">
         <string>Align to the top border of the frame</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/align-vertical-top-frame.png</normaloff>:/glmixer/icons/align-vertical-top-frame.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="distributeBox">
     <property name="title">
      <string>Distribute</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="distributeLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="distributeHorizontalLeftButton">
        <property name="toolTip">
         <string>Distribute left edges equidistantly</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/distribute-horizontal-left.png</normaloff>:/glmixer/icons/distribute-horizontal-left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="distributeHorizontalRightButton">
        <property name="toolTip">
         <string>Distribute right edges equidistantly</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/distribute-horizontal-right.png</normaloff>:/glmixer/icons/distribute-horizontal-right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="distributeHorizontalCenterButton">
        <property name="toolTip">
         <string>Distribute centers horizontally</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/distribute-horizontal-center.png</normaloff>:/glmixer/icons/distribute-horizontal-center.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="distributeVerticalBottomButton">
        <property name="toolTip">
         <string>Distribute bottom edges equidistantly</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/distribute-vertical-bottom.png</normaloff>:/glmixer/icons/distribute-vertical-bottom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="distributeVerticalGapsButton">
        <property name="toolTip">
         <string>Make vertical gaps equal</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/distribute-vertical-gaps.png</normaloff>:/glmixer/icons/distribute-vertical-gaps.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="distributeHorizontalGapsButton">
        <property name="toolTip">
         <string>Make horizontal gaps equal</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/distribute-horizontal-gaps.png</normaloff>:/glmixer/icons/distribute-horizontal-gaps.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="distributeVerticalCenterButton">
        <property name="toolTip">
         <string>Distribute centers vertically</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/distribute-vertical-center.png</normaloff>:/glmixer/icons/distribute-vertical-center.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="distributeVerticalTopButton">
        <property name="toolTip">
         <string>Distribute top edges equidistantly</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/glmixer/icons/distribute-vertical-top.png</normaloff>:/glmixer/icons/distribute-vertical-top.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
